Parsers/ASimNetworkSession/Tests/SonicWall_SonicOS_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [DvcIpAddr] of type [IP Address]: [""18C24135E8B6""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SonicOS""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 931 records (93.1%) for field [DvcAction] of type [Enumerated]: [""NA""] (Schema:NetworkSession)"
+"(0) Error: 2 invalid value(s) (up to 10 listed) in 2000 records (200.0%) for field [EventVendor] of type [Enumerated]: [""SonicWall""] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [DstFQDN] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [DstNatIpAddr] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [DstNatPortNumber] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [DvcFQDN] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [DvcOriginalAction] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [SrcFQDN] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [SrcNatIpAddr] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [SrcNatPortNumber] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [ThreatConfidence] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [ThreatIpAddr] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in optional field [ThreatOriginalConfidence] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in recommended field [DstDomain] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in recommended field [DstHostname] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in recommended field [DvcDomain] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in recommended field [DvcHostname] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in recommended field [SrcDomain] (Schema:NetworkSession)"
+"(2) Info: Empty value in 1000 records (100.0%)  in recommended field [SrcHostname] (Schema:NetworkSession)"

```

Comment:
Please see if 'recommended' values can be mapped in parser if available.

Reply:
As far as I can tell, we don't send the SourceHostName and DestinationHostName fields. They're part of CommonSecurityLog so they appear, but are empty. After some other suggested changes, these highlighted fields are not present.